### ISSUE TITLE
Excessive timewarp causes StripViewController to error 

### ISSUE THREAD

**Issue by indianbhaji** [openscope/openscope#993]
Browser: Chrome 66.0.3359.181 (Official Build) (64-bit)

Airport: all

Approximate play time before issue: after excessive timewarp command is issued (ie. timewarp 9999999)

#### Issue Description
![image](https://user-images.githubusercontent.com/19408472/40944700-a5f0939a-684d-11e8-9dec-6dc846341978.png)

I issued an excessive timewarp command, and the error pops up. I suspect that as a result of the excessive timewarp, the function is trying to check whether the aircraft is inside the airspace, when it's already been deleted

#### Steps to reproduce
-  issue an excessive timewarp command and open console

**Comment by n8rzz** [comment]
Good call out here, but yeah, this is an _extreme_ edge case.

**Comment by n8rzz** [comment]
This is an extreme edge case. closing

**Comment by erikquinn** [comment]
Do we know how high can the timewarp be set and still not have errors?

**Comment by n8rzz** [comment]
The higher the timewarp value the more unstable the app will become due to the larger changes in position each frame.  I feel like 50 is the absolute max, but even that can cause problems.  

As far as support, we should only _support_ up to `10`, as accessible via the menu.  The fact that we support a command does open the door to higher options, but I'd say that is reserved for _power users_ and wouldn't be _'officially supported'_.

**Comment by indianbhaji** [comment]
I agree that timewarp up to 10 should be supported. I've only encountered this error as I run excessive timewarps so that I can check that every airline I've put in `spawnPatterns` actually has a file and fleet defined.

**Comment by n8rzz** [comment]
And that validation is the next item on my list for `@openscope/validator`. Which will then make this use case irrelevant.

**Comment by erikquinn** [comment]
As for the timewarp thing, it's an important feature, and we should support full functionality at as high a sim rate as possible. To me, "as high as possible" tastes more like 100x, and we are currently able to handle that with the only problems being some loops due to missed fixes. So we're actually in an excellent spot in that regard. :+1: 

---

After looking at it, this is actually a _marginally_ legit bug, but not for the reasons posted. It's not related to the timewarp, though crazy timewarps result in the condition that creates the error. So here's what seems to be happening: 

If aircraft enter the airspace, their flight strip is shown in the strip bay. When they leave our airspace perimeter, the flight strip is _destroyed_ (not removed from the bay, but actually deleted). Then, if they come back into our airspace, a flight strip must not exist for them, because we are unable to find it in the `StripViewController._collection`. This situation is unlikely to be common, or real important to support, but looping errors are always best resolved if you have the opportunity, so we can keep this around as a low-priority bug. It was happening with high sim rates because a/c were missing their fixes near the boundary, entering, turning outbound to the fix, exiting, reaching the fix, turning back inbound, reenetering, and boom-- red!

---

I believe there are two solutions:

1. Never _delete_ the flight strip until the aircraft is deleted. The flight strip is _shown in the strip bay_ only when the aircraft is controllable, otherwise it is somehow hidden (provided this doesn't have performance issues with 100+ hidden strips).

1. Create a strip for the aircraft and add it to the strip bay only at the time the aircraft enters the airspace. Whenever the aircraft leaves our airspace (or is removed), delete the strip.

@n8rzz Thoughts?

**Comment by felixscheffer** [comment]
I think @erikquinn  nailed it. I can reproduce the issue without timewarp.

Airport: KBOS
![kbos-exit-retry](https://user-images.githubusercontent.com/1678164/44472607-ca726080-a62e-11e8-9242-6f1a518a2799.PNG)
